app/views/searches/_results.html.haml

```diff
@@ -1,8 +1,10 @@
 = render partial: 'results_header', locals: { term: term, dob: dob }
-
 - if results.empty?
   = render 'results/none'
 - elsif results.first.is_a? CourtDataAdaptor::Resource::ProsecutionCase
   = render partial: 'results/prosecution_case', locals: { results: results }
 - else
-  = render partial: 'results/defendant', locals: { cookies_preferences_set: cookies_preferences_set, results: results }
+  - if FeatureFlag.enabled?(:show_appeals)
+    = render partial: 'results/defendant_appeals', locals: { cookies_preferences_set: cookies_preferences_set, results: results }
```

Comment:
Do you really need to pass `cookies_pereferences_set` here? It looks to me like the old view didn't actually need it, it was just a copy-paste mistake